### DISCUSSION TITLE
Android, Usability | Add value display to precision related settings

### DISCUSSION THREAD

**Discussion by andrey6498** [ppy/osu#15444] [Ideas] [answered]
Things like offset etc use scrollers to select value and don't display it until you hover them but hovering on touch screen is not always possible which results in unintentionally changing setting screwing previous value completely. Also if touchscreen pop up with value when hovering scroller in setting should be little bit further away from "cursor" and have some kind of fixed position probably (like at least 15px top right or something)

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/issues/8377
